### PR TITLE
DDT-1111: Vessel Type for Vessel

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: datamodel/referencedata.d/vesseltypecodes.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+CONT,Container
+RORO,RoRo
+CARC,Car carrier
+PASS,Passenger
+FERY,Ferry
+BULK,Bulk
+TANK,Tanker
+LGTK,Liquified gaz tanker
+ASSI,Assistance
+PILO,Pilot boa
```
Comment (HenrikHL) [inline datamodel/referencedata.d/vesseltypecodes.csv:12]: Seems strange - seems like a copy paste error?

Comment (mp-assco) [reply datamodel/referencedata.d/vesseltypecodes.csv:12]: Seems like it. Updated.

Comment (HenrikHL) [inline datamodel/referencedata.d/vesseltypecodes.csv]:
```suggestion
PILO,Pilot boat
```

Comment (HenrikHL) [reply datamodel/referencedata.d/vesseltypecodes.csv]: It should be boat ;-)

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 2]
```diff
@@ -179,6 +179,12 @@ CREATE TABLE dcsa_im_v3_0.cut_off_time (
     cut_off_time_description varchar(250) NULL
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.vessel_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.vessel_type (
+    vessel_type_code varchar(4) PRIMARY KEY,
+    vessel_type_description varchar(100) NOT NULL
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:185]:
```suggestion
    code varchar(4) PRIMARY KEY,
    description varchar(100) NOT NULL
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 3]
```diff
@@ -190,10 +196,10 @@ CREATE TABLE dcsa_im_v3_0.vessel (
     is_dummy boolean NOT NULL default false,
     length numeric NULL,
     width numeric NULL,
+    type varchar(4) NULL REFERENCES dcsa_im_v3_0.vessel_type (vessel_type_code),
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:199]:
```suggestion
    type varchar(4) NULL REFERENCES dcsa_im_v3_0.vessel_type (code),
```

File: datamodel/referencedata.d/vesseltypecodes.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
+vessel_type_code,vessel_type_description
```
Comment (HenrikHL) [inline datamodel/referencedata.d/vesseltypecodes.csv:1]:
```suggestion
code,description
```

**Comment by HenrikHL** [comment]
I have added comments to start following one of our "new" principles - not the prefix attributes with entity name...

**Pull request merged**